UPGRADING.md

```diff
@@ -25,36 +41,56 @@ How do you know if source maps are working correctly? Try adding a syntax error
 
 ## Manifest.js
 
-Previously, if you wanted Rails to serve a non-standard named asset (any CSS not called `application.css` or JS not called `application.js`) you would have to add those files to a precompile list. For example, if you needed a marketing page with its own CSS you might add something like this:
+When compiling assets with Sprockets, Sprockets needs to decide which top-level targets to compile, usually `application.css`, `application.js`, and images.
+
+If you are using sprockets prior to 4.0, Rails will compile `application.css`, `application.js`; and *any* files found in your assets directory(ies) that are _not_ recognized as JS or CSS, but do have a filename extension. That latter was meant to apply to all your images usually in `./app/assets/images/`, but could have targetted other files as well.
+
+If you wanted to specify additional assets to deliver that were not included by this logic, for instance for a marketing page iwht its own CSS, you might add something like this:
```

Comment:
```suggestion
If you wanted to specify additional assets to deliver that were not included by this logic, for instance for a marketing page whit its own CSS, you might add something like this:
```

Reply:
oops you fixed my typo with another typo :(

Reply:
lol

Reply:
Fixed.